jax/_src/api.py

```diff
@@ -346,6 +346,13 @@ def xla_computation(fun: Callable,
                     donate_argnums: int | Iterable[int] = ()) -> Callable:
   """Creates a function that produces its XLA computation given example args.
 
+  .. warning::
+
+    This function is deprecated as of JAX v0.4.30.
```

Comment:
You should link to the Changelog for 0.4.30 too which has 1 more example. Or add that example here because that one is important.

Reply:
Done, thanks!